cms/djangoapps/contentstore/views/preview.py

```diff
@@ -247,30 +247,30 @@ def get_user_group_id_for_partition(self, user, user_partition_id):
         return None
 
 
-def _load_preview_block(request, descriptor):
+def _load_preview_block(request, block):
```

Comment:
Nit: let's add some typing if we're changing this.

```suggestion
def _load_preview_module(request: Request, block: XModuleMixin):
```

We'll need to add `from rest_framework.request import Request` and change `x_module` imports to `from xmodule.x_module import AUTHOR_VIEW, PREVIEW_VIEWS, STUDENT_VIEW, ModuleSystem, XModuleMixin`.

Reply:
Done.